src/en/getting-started.md

```diff
@@ -49,52 +57,40 @@ particular cloud provider. This is done by generating and editing the following
 ## Windows
     %LOCALAPPDATA%/Juju
```

Comment:
Note: this file (getting-started) is bundled into the Windows installer (as a README) and pops up upon installation.